src/nodes/html.ts

```diff
@@ -983,6 +992,10 @@ export interface Options {
 	lowerCaseTagName: boolean;
 	comment: boolean;
 	parseNoneClosedTags?: boolean;
+	voidTag?: {
+		closingSlash?: boolean;
+		closingSpace?: 'never' | 'always' | 'attrPresent';
```

Comment:
Will `attrPresent` only be easier to use?

Reply:
`attrPresent` with `never` and `always` values allow for fine grained formatting. I have discovered with test units (not included here) some cases where spaces were removed.